pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -179,6 +179,11 @@ func (p *LaunchTemplateProvider) createLaunchTemplate(ctx context.Context, optio
 	output, err := p.ec2api.CreateLaunchTemplateWithContext(ctx, &ec2.CreateLaunchTemplateInput{
 		LaunchTemplateName: aws.String(launchTemplateName(options)),
 		LaunchTemplateData: &ec2.RequestLaunchTemplateData{
+			BlockDeviceMappings: []*ec2.LaunchTemplateBlockDeviceMappingRequest{{
```

Comment:
Yep, and so will VolumeSize 

```suggestion
			BlockDeviceMappings: []*ec2.LaunchTemplateBlockDeviceMappingRequest{{
				Ebs: &ec2.LaunchTemplateEbsBlockDeviceRequest{
					Encrypted:  aws.Bool(true),
					VolumeSize: aws.Int64(20),
				},
				DeviceName: aws.String("/dev/sdf"),
			}},
```